src/kili/adapters/kili_api_gateway/organization/operations_mixin.py

```diff
@@ -0,0 +1,108 @@
+"""Mixin extending Kili API Gateway class with Organization related operations."""
+
+from typing import Dict, Generator, Optional
+
+import tqdm
+
+from kili.adapters.kili_api_gateway.base import BaseOperationMixin
+from kili.adapters.kili_api_gateway.helpers.queries import (
+    PaginatedGraphQLQuery,
+    QueryOptions,
+    fragment_builder,
+)
+from kili.adapters.kili_api_gateway.organization.mappers import (
+    map_organization_data,
+    map_organization_metrics_where,
+    map_organization_update_data,
+    map_organization_where,
+)
+from kili.adapters.kili_api_gateway.organization.types import (
+    KiliAPIGateWayCreateOrganizationInput,
+    KiliAPIGateWayUpdateOrganizationInput,
+)
+from kili.domain.organization import (
+    Organization,
+    OrganizationFilters,
+    OrganizationId,
+    OrganizationMetricsFilters,
+)
+from kili.domain.types import ListOrTuple
+
+from .operations import (
+    ORGANIZATION_FRAGMENT,
+    get_count_organizations_query,
+    get_create_organization_mutation,
+    get_list_organizations_query,
+    get_organization_metrics_query,
+    get_update_properties_in_organization,
+)
+
+
+class OrganizationOperationMixin(BaseOperationMixin):
+    """GraphQL Mixin extending GraphQL Gateway class with Organization related operations."""
+
+    def create_organization(
+        self,
+        organization: KiliAPIGateWayCreateOrganizationInput,
+        description: str,
+        disable_tqdm: Optional[bool],
+    ) -> Organization:
+        """Send a GraphQL request calling createOrganization resolver."""
+        with tqdm.tqdm(total=1, desc=description, disable=disable_tqdm) as pbar:
+            payload = map_organization_data(organization)
+            result = self.graphql_client.execute(
+                get_create_organization_mutation(ORGANIZATION_FRAGMENT), payload
+            )
+            pbar.update(1)
+
+        return result["data"]
```

Comment:
not sure it's useful to spawn a progress bar just for one item

we should stay consistent with the other methods that currently don't do this